eng/spotbugs-aggregate-report/pom.xml

```diff
@@ -68,8 +68,8 @@
                 <source>..\..\sdk\storage\azure-storage-blob-cryptography\src\samples\java\com</source>
                 <source>..\..\sdk\storage\azure-storage-file-share\src\main\java\com</source>
                 <source>..\..\sdk\storage\azure-storage-file-share\src\samples\java\com</source>
-<!--                <source>..\..\sdk\storage\azure-storage-file-datalake\src\main\java\com</source>-->
-<!--                <source>..\..\sdk\storage\azure-storage-file-datalake\src\samples\java\com</source>-->
+                <source>..\..\sdk\storage\azure-storage-file-datalake\src\main\java\com</source>
```

Comment:
This is going to hit an issue with SpotBugs if we don't bump the version of Azure Core Datalake is depending on.

Reply:
@alzimmermsft  You're right. All of the changes made have to be reverted including the source changes for the core changes post GA which were changed back in the original PR.